### ISSUE TITLE
Add support for hooking in validators

### ISSUE THREAD

**Issue by joeljames** [localmed/pyserializer#4]
Add support for validating fields when deserializing a python object.
eg: 

```
class UserDeserializer(Serializer):
    email = fields.CharField(
        validators=[validators.RequiredValidator()]
    )
    username = fields.CharField()

    class Meta:
        fields = (
            'email',
            'username'
        )

    def __repr__(self):
        return '<User(%r)>' % (self.username)

input_data = {
    'username': 'JohnSmith'
}
UserDeserializer(data_dict=input_data)
deserializer.errors
```

**Comment by joeljames** [comment]
https://github.com/localmed/pyserializer/pull/5

**Comment by joeljames** [comment]
Released 0.2.0 with support for validation.
